metrics_api/.rubocop.yml

```diff
@@ -15,8 +16,14 @@ Metrics/CyclomaticComplexity:
   Max: 20
 Metrics/ParameterLists:
   Enabled: false
+
+Naming/VariableNumber:
+  Enabled: false
 Naming/FileName:
   Exclude:
     - "lib/opentelemetry-metrics-api.rb"
+
+Style/IfUnlessModifier:
+  Enabled: false
```

Comment:
nit: When this is enabled, we can be forced into writing long lines with `if/unless` at the end and I think may not be easy to read